force-app/volunteers/labels/CustomLabels.labels

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomLabels xmlns="http://soap.sforce.com/2006/04/metadata">
+    <labels>
+        <fullName>emailVolunteersTitle</fullName>
+        <categories>Volunteers</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Email volunteers toast title</shortDescription>
+        <value>Email Volunteers</value>
```

Comment:
It occurs to me that we're tracking status on the assignments, so as a volunteer manager I would expect that only volunteers who are actively assigned to the position/activity/etc. would appear in the email list, or I would have a way to specify whether to include them, or I could at least see who was inactive and have a way to manually deselect inactive folks. I recognize the complexity of the ask, so we can punt that to a future issue.

Reply:
I have added a filter to the volunteer assignment service which will only return active volunteers.  This change is in ff95892.

Adding a more sophisticated UI to allow users to control this behavior or select volunteers more interactively is in the feature backlog as #68.  Note that it might be worth considering whether the email button should exist on the detail page for the position/activity/shift or if it should be a list action button instead, which could give users more nuanced control over which volunteers to include the email.